controls/scheduler/built-in-dialogs.md

```diff
@@ -0,0 +1,114 @@
+
+![Telerik .NET MAUI Scheduler DeleteRecurrenceChoiceDialog](images/scheduler-dialogs-deleterecurrencechoice.png)
+
+#### Delete Appointment Choice Dialog
+
+(Desktop-only) When the end user attempts to delete an appointment, asks for confirmation.
+
+![Telerik .NET MAUI Scheduler DeleteAppointmentChoiceDialog](images/scheduler-dialogs-deleteappointmentchoice.png)
+
+## Methods
+
+Use the following methods to show the built-in dialogs of the Scheduler:
+
+* `CreateAppointmentWithDialog(DataRange range)`&mdash;Opens a dialog for creating an appointment.
+* `EditAppointmentWithDialog(Occurrence occurrence)`&mdash;Opens a dialog for editing an exsisting appointment.
+* `DeleteAppointmentWithDialog(Occurrence occurrence)`&mdash;Opens a dialog for deleting an existing appointment.
+
+The following example demonstrates how to use the methods for managing the built-in dialogs:
+
+**1.** Create a sample view with a `RadScheduler` instance and a few buttons:
+
+<snippet id='scheduler-dialogs-methods-xaml' />
+
+**2.** Add the buttons' clicked event handlers in the code-behind:
+
+<snippet id='scheduler-dialogs-methods-eventhandlers' />
+
+## Events
+
+The Scheduler exposes the following events for its built-in dialogs:
+
+* `DialogOpening`&mdash;Raised when a dialog is about to be opened. The `DialogOpening` event handler receives two parameters:
+    * The sender argument, which is of type `object`, but can be cast to the `RadScheduler` type.
+    * A `SchedulerDialogOpeningEventArgs` object with the following properties:
+       * `DialogType`&mdash;Provides the dialog type.
+       * `Cancel`&mdash;If `True`, cancels the dialog opening.
+* `DialogClosing`&mdash;Raised when a dialog is about to be closed. The `DialogClosing` event handler receives two parameters:
+    * The sender argument, which is of type `object`, but can be cast to the `RadScheduler` type.
+    * A `SchedulerDialogClosingEventArgs` object with the following properties:
+       * `DialogType`&mdash;Provides the dialog type.
+       * `Cancel`&mdash;If `True`, cancels the dialog closing.
+       * `DialogResult (bool?)`&mdash;Indicates how the dialog was closed. `null` when closed with the **Close** button. `False` when closed with the **Cancel** button. `True` when <BE SPECIFIC>.
+
+Check a quick example on how to use the `DialogOpening` and `DialogClosing` events:
+
+**1.** Add the `RadScheduler` definition:
+
+<snippet id='scheduler-dialogs-events-xaml' />
+
+**2.** Add the event handlers:
+
+<snippet id='scheduler-dialogs-events' />
+
+## Customization
+
+You can style the visual appearance of each dialog by creating the corresponding `Style` with `TargetType` set to `SchedulerDialog`. Here is a list of the available `Style` properties:
+
+* `EditAppointmentDialogStyle`&mdash;Defines the `Style` that will be applied on the Edit Appointment dialog.
```

Comment:
I probably missed this the first time around (sorry for that). Go with "Defines the `Style` for the " for all bullets :)